js/popup.js

```diff
@@ -0,0 +1,12 @@
+const _popupContent = `
+<div class="popup">
+  <div class="option">Block this user</div>
+  <div class="divider"></div>
+  <div class="option">Restrict</div>
+  <div class="divider"></div>
+  <div class="option">Report User</div>
+  <div class="divider"></div>
+  <div class="option cancel">Cancel</div>
+</div>
+`;
+export default _popupContent;
```

Comment:
Add an empty line at the end of your file.

Reply:
Alright